server/channels/jobs/base_workers.go

```diff
@@ -50,9 +50,6 @@ func (worker *SimpleWorker) Run() {
 			worker.logger.Debug("Worker received stop signal")
 			return
 		case job := <-worker.jobs:
-			job.Logger = job.Logger.With(mlog.String("workername", worker.name))
-
-			job.Logger.Debug("Worker received a new candidate job")
```

Comment:
This is now logged from within `DoJob` where we have to apply the interesting fields to the logger anyway.